File: 백준/Bronze/2530. 인공지능 시계/인공지능 시계.py
a,b,c = map(int,input().split())
n = int(input())
c += n
b += c//60
c = c%60
a += b//60
b = b%60
if a >= 24:
    cnt = a//24
    a -= 24*cnt
print(a,b,c)

File: 백준/Bronze/2530. 인공지능 시계/인공지능 시계.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        int se = sc.nextInt();
        int cnt = (a*3600) + (b*60) + c + se;
        if (cnt/3600 > 23) {
            System.out.print((cnt/3600) % 24 + " ") ;
            System.out.print((cnt%3600)/60 + " ");
            System.out.print((cnt%3600)%60);
        }
        else {
            System.out.print((cnt/3600) + " ") ;
            System.out.print((cnt%3600)/60 + " ");
            System.out.print((cnt%3600)%60);
        }
    }
}